<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="massLabel">
       <property name="text">
        <string>Mass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="massEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="diameterLabel">
       <property name="text">
        <string>Diameter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="diameterEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="muzzleVelocityLabel">
       <property name="text">
        <string>Muzzle Velocity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="muzzleVelocityEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="windSpeedLabel">
       <property name="text">
        <string>Wind Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="windSpeedEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QComboBox" name="unitComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Metric</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imperial</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="windSpeedLabel_2">
       <property name="text">
        <string>Launch Angle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="launchAngleEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="windDirectionLabel">
       <property name="text">
        <string>Wind Direction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="windDirectionEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="latitudeLabel">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="latitudeEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="dragCoeffLabel">
       <property name="text">
        <string>Drag Coefficent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dragCoeffEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
